### ISSUE TITLE
Is this repo abandonware? Should I switch to ar.js?

### ISSUE THREAD

**Issue by dogzilla** [hiukim/mind-ar-js#526]
While I love the features of mindar, the lack of responsiveness or community around this project makes me question how smart it is to use this project as a foundation. It kinda seems like there's little interest in it. Am I missing a useful resource somewhere for actual support or communication with someone who can speak to the innards of this project, or should I simply go with a more popular alternative?

**Comment by CoderSilas** [comment]
Hello,
i am just testing things and use the quick guide from the documentation. Its not running because it uses legacylights and one script wasnt hosted anymore.
I think its fair to say that its currently not really maintained. 
EDIT: Just saw some commits for version updates, 4 months ago. So its alive but no exiting new features. AR.js on the other hand is definitively dead if you dig around a bit.

**Comment by dariocavada** [comment]
Has anyone found a maintained alternative?

**Comment by subramani01** [comment]
@CoderSilas @dariocavada have you guys found any alternatives to mindar and ar.js?

**Comment by dogzilla** [comment]
Not any viable ones. We just forked it and started hacking on it. I think that’s probably the best bet - it’s a good start, but I gotta imagine that there’s lots of potential improvements since it was written.

**Comment by CoderSilas** [comment]
Yeah I found one, called encantar.js. That's the one I am currently using which is updated regularly while I wait for the proper and official solutions from Google or Mozilla..

**Comment by dogzilla** [comment]
I took a look at it - found the jitter to be kinda bad and I couldn’t find any params to control that like in MindAR. Is encantar worth a second look? What do you like about it?Sent from my iPhoneOn Dec 15, 2024, at 6:05 AM, CoderSilas ***@***.***> wrote:﻿
Yeah I found one, called encantar.js. That's the one I am currently using which is updated regularly while I wait for the proper and official solutions from Google or Mozilla..

**Comment by CoderSilas** [comment]
Well, I wish there was a proper solution but it's easier to use and updated unlike mindar..

**Comment by dogzilla** [comment]
I have a question for the group:

I’d love to see this mailing list grow into a community and mutual resource, for MindAR devs and just general webAR dev. I feel like the mailing list format is limiting. A subreddit would be good, but Reddit seems to be going through some things as a company and service, so I’ve been looking at Lemmy - an open source social network that seems inspired by Reddit (https://github.com/amirzaidi/lemmy). 

Would the group be interested in joining if I set up a Lemmy community for us?

**Comment by dariocavada** [comment]
> @CoderSilas @dariocavada have you guys found any alternatives to mindar and ar.js?

Unfortunately, I haven't found anything else of interest, only commercial products that we are considering for integration, namely Zappar and Immersal (https://immersal.com/webar), to offer advanced AR interactions. However, having the right client is essential in this case. It would be great to have a fully open-source system, but significant investments are needed to build something that works well.

**Comment by dogzilla** [comment]
Agreed. We’ve found the same, and while the features for eg: 8thWall are pretty good, the licensing terms and prices are frankly too onerous for the majority of our projects. So we’ve started hacking MindAR to develop our own features, eventually hopefully comparable to those offered by commercial services. We’ve had some success in this already, with admittedly simpler functionality. But it gives us hope that we can tackle the more complex features, such as SLAM functionality, per-object persistence, the ability to use prerecorded video instead of the camera as input to the AR system, and alternative tracking models to reduce jitter and improve low-light functionality. 

We’re happy to work with other contributors to create something that the entire community can benefit from.

**Comment by CoderSilas** [comment]
Would be great to have this project moving forward with developers but i am not a webdev. Not sure if @hiukim will pick it up anytime soon but the technology itself is super fun to play with. With google having a WebXR image tracker in chrome behind a flag which works perfectly for years is a shame too.

So i dont know, i can only wait for a new solution and if possible help with some money, did it here but i guess thats not the needed motivation. 

Also there are a discord servers about WebXR.. would not open another new community for that.

**Comment by amcc** [comment]
i'm really keen to know of any updates too @dogzilla if you do have anything to share that would be fantastic, i can try to help if i'm able to - more of a front end guy, but might be able to get help from others too. Right now i'm using MindAR tweaked to use updated threeJS locally due to the lack of development - fantastic project but the jitter and other issues are bugging me too.

**Comment by dogzilla** [comment]
I haven’t seen or done anything substantive, but if there’s interest I’d be happy to organize an online meet to discuss forking and updating this project.Seems like there’s a fair number of us who rely on MindAR. I have a list of desired features, and while not the best coder, I’ll do what I can, and I can write docs, create art & assets, and fetch water when needed.- DogzillaSent from my iPhoneOn Mar 23, 2025, at 5:50 PM, amcc ***@***.***> wrote:﻿
  i'm really keen to know of any updates too @dogzilla if you do have anything to share that would be fantastic, i can try to help if i'm able to - more of a front end guy, but might be able to get help from others too. Right now i'm using MindAR tweaked to use updated threeJS locally due to the lack of development - fantastic project but the jitter and other issues are bugging me too.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>



amcc left a comment (hiukim/mind-ar-js#526)
i'm really keen to know of any updates too @dogzilla if you do have anything to share that would be fantastic, i can try to help if i'm able to - more of a front end guy, but might be able to get help from others too. Right now i'm using MindAR tweaked to use updated threeJS locally due to the lack of development - fantastic project but the jitter and other issues are bugging me too.
